controllers/apps/transformer_component_deletion.go

```diff
@@ -169,6 +170,7 @@ func (t *componentDeletionTransformer) getCluster(transCtx *componentTransformCo
 func compOwnedKinds() []client.ObjectList {
 	return []client.ObjectList{
 		&workloads.ReplicatedStateMachineList{},
+		&policyv1.PodDisruptionBudgetList{},
```

Comment:
and remove PDB from cluster controller?

Reply:
add comments: [a288947](https://github.com/apecloud/kubeblocks/pull/6985/commits/a288947f67d4790047b766bd3bd43edd7bb35109)